### DISCUSSION TITLE
public/.htaccess is missing in pimcore 10.5.2

### DISCUSSION THREAD

**Discussion by GaneshKandu** [pimcore/pimcore#12828] [Q&A / Support] [answered]
### Expected behavior

Not an internal server error.

### Actual behavior

I am getting this error in apache log while accessing /admin of pimcore 10.5.2 install

[Fri Aug 05 02:21:06 2022] [error] [client 192.168.1.111] Request exceeded the limit of 10 internal redirects due to probable configuration error. Use 'LimitInternalRecursion' to increase the limit if necessary. Use 'LogLevel debug' to get a backtrace.

.htaccess is missing from public folder it was in 10.5.0

I copied .htaccess of 10.5.0 to 10.5.2 and pimcore 10.5.2 install started working.

### Steps to reproduce

Install Pimcore 10.5.2

**Comment by GaneshKandu** [comment]
I am using Apache
admin link is /admin there is no admin folder in public folder of pimcore
am getting 404 error for admin
only / is working.
Laravel web debug toolbar's ajax calls are also 404 error.

**Comment by kingjia90** [comment, marked as answer]
It's not a bug, `.htaccess` is not shipped anymore out of the box in the latests releases, please see:
https://github.com/pimcore/skeleton/pull/101
https://github.com/pimcore/demo/pull/326
it's documented here https://github.com/pimcore/pimcore/pull/12619

It's now up to the developer and depending on the webserver

**Reply by dvesh3** [reply]
Moved to discussions as it could help someone else understanding the situation.
